mpReview.py

```diff
@@ -896,9 +917,13 @@ def setupConnections(self):
 
   def checkAndSetLUT(self):
```

Comment:
For the future - it would be good to validate that the selected JSON file in fact contains the expected content!

Reply:
Good point, I included some basic checks to see if certain fields exist in the terminology json file. If any fail, a warning is printed, and the default (prostate) one is used instead.